foodnet/membership/tests.py

```diff
@@ -35,15 +47,15 @@ def setUp(self):
         email_address.save()
 
     def test_profile(self):
-        resp = self.client.get(reverse('profile'))
+        response = self.client.get(reverse('profile'))
```

Comment:
Yes `response` is much more readable than `resp` in my opinion, even though it's only 4-5 lines of code :)

If I didn't refactor the tests, they would fail after refactoring the models ;)

Of course I want us to adopt a style guide which I feel is the right one, anything else would be weird :P But what I want even more is us to have a common style on which we all can agree. This would mean that we wouldn't step on each others toes as much (as for instance I have done here).

If you wish, I can change it back to `resp` :)